osu.Game/Rulesets/Difficulty/DifficultyAttributes.cs

```diff
@@ -31,9 +31,7 @@ public class DifficultyAttributes
         protected const int ATTRIB_ID_NESTED_SCORE_PER_OBJECT = 37;
         protected const int ATTRIB_ID_LEGACY_SCORE_BASE_MULTIPLIER = 39;
         protected const int ATTRIB_ID_MAXIMUM_LEGACY_COMBO_SCORE = 41;
-        protected const int ATTRIB_ID_MECHANICAL_DIFFICULTY = 43;
         protected const int ATTRIB_ID_RHYTHM_DIFFICULTY = 45;
-        protected const int ATTRIB_ID_READING_DIFFICULTY = 47;
         protected const int ATTRIB_ID_CONSISTENCY_FACTOR = 49;
```

Comment:
Shouldn't those be moved to 43 and 45 respectively?

Reply:
Done